dev-tools/globals.ts

```diff
@@ -1006,3 +1006,10 @@ interface RandomTeamsTypes {
 		fillerMoves4?: string[]
 	}
 }
+
+interface PokemonModData {
+	gluttonyFlag?: boolean; // Gen-NEXT
+	innate?: string; // Gen-NEXT
```

Comment:
this is used for Partners in Crime, not Gen-NEXT

Reply:
Thanks, fixed!